tailor_image/create_image.py

```diff
@@ -141,10 +142,25 @@ def create_image(name: str, distribution: str, apt_repo: str, release_track: str
 
     elif build_type == 'ami':
         image_name = f'{organization}_{name}_{distribution}_ami_{release_label}'
+        # Get ami-id for base image
+        ec2 = boto3.client('ec2')
+        source_image_name = recipe[name]['source_ami'].replace('$distribution', distribution)
+        source_ami_id = None
+        images = ec2.describe_images(Owners=['self'])['Images']
+        for image in images:
```

Comment:
this is a linear search through all our AMIs. is this going to get painful as our list of AMIs grow? Do we prune AMIs currently?

Reply:
Is there a 'find' AMI type method?

Reply:
we overwrite the sim images, so we only keep the latest for each release/feature atm